public/app/features/alerting/unified/RuleEditorExisting.test.tsx

```diff
@@ -137,7 +114,36 @@ describe('RuleEditor grafana managed rules', () => {
     // save and check what was sent to backend
     await user.click(ui.buttons.save.get());
 
-    mocks.searchFolders.mockResolvedValue([] as DashboardSearchHit[]);
     expect(screen.getByText('New folder')).toBeInTheDocument();
   });
+
+  it('saves evaluation interval correctly', async () => {
```

Comment:
It was more fiddly than you might think to get this test working - there were a few places where we were passing invalid DOM props through to elements and were getting console.errors, and I had to tweak the folders API responses to make it work. 

The good thing is that the mock API enabled in browser makes it a bit easier to work out what's going on (it looks like this, which reflects our mock data)
![image](https://github.com/user-attachments/assets/adc91832-c285-4982-a404-099f27d1d334)

I had to change a test elsewhere so we can use a different folder/group combination